templates/page.tmpl

```diff
@@ -124,7 +123,7 @@
 <img alt="" border="0" src="https://www.paypal.com/en_US/i/scr/pixel.gif" width="1" height="1">
 </form>
 <a class="FlattrButton" style="display:none;" rev="flattr;button:compact;"
-href="http://awesome.naquadah.org"></a>
+href="http://awesomewm.org"></a>
```

Comment:
@jd ?
IIRC there is also some bitcoin address?!
The domain for example costs ~15€/year. Do we have other expenses?